ui/tests/acceptance/stepDefinitions/accountsContext.js

```diff
@@ -12,10 +12,22 @@ Then('user {string} should be displayed in the accounts list on the WebUI', asyn
   return assert.strictEqual(userListed, username)
 })
 
+Given('the user has changed the role of user {string} to {string}', function (username, role) {
+  return client.page.accountsPage().selectRole(username, role)
+})
+
 When('the user changes the role of user {string} to {string} using the WebUI', function (username, role) {
   return client.page.accountsPage().selectRole(username, role)
 })
 
 Then('the displayed role of user {string} should be {string} on the WebUI', function (username, role) {
   return client.page.accountsPage().checkUsersRole(username, role)
 })
+
+Then('the user should not be able to see the accounts list on the WebUI', async function () {
+  return client.page
+    .accountsPage()
+    .waitForElementVisible('@loadingAccountsList')
+    .expect.element('@accountsListTable')
```

Comment:
This should involve some waiting, right? The loader is an intermediate state for an admin user as well, so we should have like a 5 seconds delay before we check if the accounts list was loaded.

Reply:
Yeah, you are right. I have added 5 seconds timeout now before checking for accounts list.